chatterbox/src/Auth/Register/stepone.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { TextField } from 'material-ui';
+
+class StepOne extends Component {
+    constructor(e) {
+        super(e);
+        this.state = {
+            name: '',
+            email: '',
+            password: ''
+        };
+        this.checkCanNext = this.checkCanNext.bind(this);
+    }
+    handleName(e) {
+        console.log('checkCanNext:', this.checkCanNext());
+        this.setState({ name: e.target.value });
+    }
+    handleEmail(e) {
+        console.log('checkCanNext:', this.checkCanNext());
+        this.setState({ email: e.target.value });
```

Comment:
again remove console.log and setstate should be in not be in the code, use a reducer